lib/checks/label/label-content-name-mismatch.js

```diff
@@ -0,0 +1,34 @@
+function getAriaText(elm) {
+	if (elm.getAttribute('aria-label')) {
+		return elm
+			.getAttribute('aria-label')
+			.toLowerCase()
+			.trim();
+	}
+	if (elm.getAttribute('aria-labelledby')) {
+		return elm
+			.getAttribute('aria-labelledby')
+			.toLowerCase()
+			.trim();
+	}
+	return null;
+}
+
+const accessibleText = getAriaText(node);
+// if no accessible text - fail
+if (!accessibleText || !accessibleText.length) {
+	return false;
+}
+
+const contentText = node.textContent.toLowerCase().trim();
+// if no text content - fail
+if (!contentText || !contentText.length) {
+	return false;
+}
+
+// if text content is not part of accessible text - fail
+if (!accessibleText.includes(contentText)) {
```

Comment:
I don't know that we have a string polyfill of `.includes`. Make sure this works on IE.

Reply:
We seem to have a polyfill for `String.prototype.includes` - https://github.com/dequelabs/axe-core/blob/develop/lib/core/utils/pollyfills.js#L288